src/Components/RecentJobs/RecentBox/index.jsx

```diff
@@ -0,0 +1,40 @@
+import { useEffect, useState } from "react"
+import SingleJob from "./SingleJob"
+
+function RecentBox() {
```

Comment:
I would suggest renaming this component to make it's intention a little bit clearer, something like RecentJobsTable would be better